### ISSUE TITLE
Jigzi game: Training for leadership in the idf

### ISSUE THREAD

**Issue by chayaml** [chayaml/Israel-bytes#80]
Youtube video cant be played

<img width="1242" alt="Screen Shot 2022-11-30 at 12 20 45 PM" src="https://user-images.githubusercontent.com/78186896/204865625-f6c63c5c-7ece-46f8-b835-3b01610049bd.png">

https://jigzi.org/asset/play/jig/322cdd02-6473-11ed-9130-53b18f194a26?direction=ltr&display_score=false&track_assessments=false&drag_assist=false&is_student=false&draft_or_live=live

**Comment by yechielhoffman** [comment]
I tried it in my browser and it played.

**Comment by chayaml** [comment]
@yechielhoffman you're right I tried it again and it worked - I'm not sure though why I cant pause/ control the vid. Will ask the dev team
